packages/sdk/src/Actions/EventAction.ts

```diff
@@ -516,6 +519,29 @@ export class EventAction extends DeployableTarget<
     params: ValidateActionStepParams,
   ): Promise<Address | undefined> {
     const signature = claimant.signature;
+
+    // find message sender and return it
+    // WARNING: this is error prone in bridged transactions, as from will be exit node
+    if (signature === zeroHash || claimant.fieldIndex === 255) {
```

Comment:
I think `claimant.fieldIndex === 255` wll be more consistent since `signature === zeroHash` will be default state. Unless there's a reason we need to support both. Feels less error prone.